fe/src/main/java/org/apache/doris/load/loadv2/etl/EtlJobConfig.java

```diff
@@ -0,0 +1,263 @@
+// under the License.
+
+package org.apache.doris.load.loadv2.etl;
+
+import com.google.common.collect.Lists;
+
+import java.util.List;
+import java.util.Map;
+
+/** jobconfig.json file format
+ * {
+ * 	"tables": {
+ * 		10014: {
+ * 			"columns": {
+ * 				"k1": {
+ * 					"default_value": "\\N",
+ * 					"column_type": "DATETIME",
+ * 					"is_allow_null": true
+ *              },
+ * 				"k2": {
+ * 					"default_value": "0",
+ * 					"column_type": "SMALLINT",
+ * 					"is_allow_null": true
+ *              },
+ * 				"v": {
+ * 					"default_value": "0",
+ * 					"column_type": "BIGINT",
+ * 					"is_allow_null": false
+ *              }
+ *          },
+ * 			"indexes": {
+ * 				10014: {
+ * 					"column_refs": [{
+ * 						"name": "k1",
+ * 						"is_key": true,
+ * 						"aggregation_type": "NONE"
+ *                    }, {
+ * 						"name": "k2",
+ * 						"is_key": true,
+ * 						"aggregation_type": "NONE"
+ *                    }, {
+ * 						"name": "v",
+ * 						"is_key": false,
+ * 						"aggregation_type": "NONE"
+ *                    }],
+ *                  "distribution_column_refs": ["k1"],
+ * 					"schema_hash": 1294206574
+ *              },
+ * 				10017: {
+ * 					"column_refs": [{
+ * 						"name": "k1",
+ * 						"is_key": true,
+ * 						"aggregation_type": "NONE"
+ *                    }, {
+ * 						"name": "v",
+ * 						"is_key": false,
+ * 						"aggregation_type": "SUM"
+ *                    }],
+ *                  "distribution_column_refs": ["k1"],
+ * 					"schema_hash": 1294206575
+ *                }
+ *           },
+ * 			"partition_info": {
+ * 				"partition_type": "RANGE",
+ * 				"partition_column_refs": ["k2"],
+ * 				"partitions": {
+ * 					10020: {
+ * 						"start_keys": [-100],
+ * 						"end_keys": [10],
+ * 						"is_max_partition": false,
+ * 						"bucket_num": 3
+ *                  }
+ *              }
+ *          },
+ * 			"file_groups": [{
+ * 		        "partitions": [10020],
+ * 				"file_paths": ["hdfs://hdfs_host:port/user/palo/test/file"],
+ * 				"file_field_names": ["tmp_k1", "k2"],
+ * 				"value_separator": ",",
+ * 			    "line_delimiter": "\n"
+ * 				"column_mappings": {
+ * 					"k1": {
+ * 						"function_name": "strftime",
+ * 						"args": ["%Y-%m-%d %H:%M:%S", "tmp_k1"]
+ *                   }
+ *              },
+ * 				"where": "k2 > 10",
+ * 				"is_negative": false,
+ * 				"hive_table_name": "hive_db.table"
+ *          }]
+ *      }
+ *  },
+ * 	"output_path": "hdfs://hdfs_host:port/user/output/10003/label1/1582599203397",
+ * 	"output_file_pattern": "label1.%(table_id)d.%(partition_id)d.%(index_id)d.%(bucket)d.%(schema_hash)d"
+ * }
+ */
+public class EtlJobConfig {
+    public static final String GLOBAL_DICT_TABLE_NAME = "doris_global_dict_table_%d";
+    public static final String DISTINCT_KEY_TABLE_NAME = "doris_distinct_key_table_%d_%s";
+    public static final String DORIS_INTERMEDIATE_HIVE_TABLE_NAME = "doris_intermediate_hive_table_%d_%s";
+
```

Comment:
This GSON naming policy is `LOWER_CASE_WITH_UNDERSCORES`. But here you use CamelCase. Does it matter?